tests/shared/src/test/scala-2.13/scala/meta/tests/parsers/dotty/Scala3PositionSuite.scala

```diff
@@ -0,0 +1,117 @@
+package scala.meta.tests.parsers.dotty
+
+import scala.meta._
+import scala.meta.tests.parsers.BasePositionSuite
+
+class Scala3PositionSuite extends BasePositionSuite {
+  override def defaultDialect: Dialect = dialects.Scala3
+
+  check[Type]("A & B")
+  check[Type]("A | B")
+  check[Type](
+    "[X] =>> (X, X)",
+    """|Type.Bounds [X→←] =>> (X, X)
```

Comment:
```suggestion
    """|Type.Bounds [X@@] =>> (X, X)
```
Or something it's kind of hard to see in the git diff.

![image](https://user-images.githubusercontent.com/3807253/102233025-175d5d80-3ef0-11eb-944a-729065f6e147.png)

Reply:
Good idea, done.